### PR TITLE
PD-32165: TEST token migration without dependency of installing Fonteva Payments

### PR DESCRIPTION
**Pull request by jupadhyaya**

### CODE REVIEW

File: force-app/main/default/classes/TokenMigrationBatch.cls [hunk 1]
```diff
@@ -104,47 +102,25 @@ global with sharing class TokenMigrationBatch implements Database.Batchable<sObj
...
             }
-
-            if (!paymentMethodsToInsert.isEmpty()) {
-                Framework.SObjectService.insertRecords(paymentMethodsToInsert);
-            }
-
             SpreedlyHTTPService.insertSystemLogs();
         } catch(Exception e) {
-            errorSystemLogs.add(SpreedlyHTTPService.createSystemLog('Spreedly Error Occured', e.getMessage() + ' ' + e.getStackTraceString()));
+            errorSystemLogs.add(SpreedlyHTTPService.createSystemLog('Spreedly Error Occure', e.getMessage() + ' ' + e.getStackTraceString()));
```
Comment (VikasPoddar1319) [inline force-app/main/default/classes/TokenMigrationBatch.cls]: @jupadhyaya Please correct the spelling of Occured!

File: force-app/main/default/Objects/fields/Token_Migration_Test__c.field-meta.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (VikasPoddar1319) [inline force-app/main/default/Objects/fields/Token_Migration_Test__c.field-meta.xml]: @jupadhyaya 
Please move to the correct folder!
You can check the folder in Fonteva Payments

**Review by VikasPoddar1319 — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/TokenMigrationBatch.cls (hunk 1 above)
Comment (jupadhyaya) [reply force-app/main/default/classes/TokenMigrationBatch.cls]: updated

File: force-app/main/default/Objects/fields/Token_Migration_Test__c.field-meta.xml (hunk 2 above)
Comment (jupadhyaya) [reply force-app/main/default/Objects/fields/Token_Migration_Test__c.field-meta.xml]: updated

**Review by VikasPoddar1319 — APPROVED** [review]

**Pull request merged**